www/base/src/components/MultipleWorkersActionsModal/MultipleWorkersActionsModal.tsx

```diff
@@ -38,23 +39,43 @@ const workerToSelectOption = (worker: Worker): SelectOption => {
 }
 
 const Control = (props: any) => {
-  const { useRegexSearch, handleToggleClick, selectRef } = props;
-
-  const iconStyle = {
-    backgroundColor: useRegexSearch ? "lightgrey" : "",
-    display: "flex",
-    alignItems: "center",
-    justifyContent: "center",
-    marginRight: "5px",
-    cursor: "pointer"
-  };
+  const { useRegexSearch, handleToggleClick, handleSelectAllClick, selectRef } = props;
+
+  const [ isHoveredRegexSearch, setIsHoveredRegexSearch ] = useState(false);
+
+  const [isHoveredSelectAll, setIsHoveredSelectAll] = useState(false);
+  const [isActiveSelectAll, setIsActiveSelectAll] = useState(false);
 
   return (
     <components.Control {...props}>
-      <div style={{ display: "flex", alignItems: "center", width: "100%" }}>
+      <div className="flex-container">
```

Comment:
Probably should be named `bb-workers-actions-modal-control-container`. `flex-container` is too generic and may clash with other libraries.